src/app/auth/features/user-manager-detail/user-manager-detail.component.spec.ts

```diff
@@ -92,6 +131,8 @@ describe('UserManagerDetailComponent', () => {
         CommonDatePipe,
```

Comment:
```suggestion
        provideNoopAnimations(),
        provideRouter([
          { path: 'user-manager/doe@test.dev', component: UserManagerDetailComponent },
        ]),
        CommonDatePipe,
```

You can then remove the NoopAnimationModule, the mockRouter and the MockActivateRoute.